### ISSUE TITLE
[Backend] Route for inviting admin

### ISSUE THREAD

**Issue by jackfrost13** [HITK-TECH-Community/Community-Website#478]
The task is to add a route will which take a list of admin data for inviting them to become an admin.

This depends on #152

**Comment by Itsaadarsh** [comment]
Hey there @jackfrost13, I'd like to work on this feature. So if it's not already assigned to anyone I'd like you it assign it to me so that I could get started right away.

**Comment by Devenkapoor7303** [comment]
Can I work on this issue as a part of GSSOC 21

**Comment by jackfrost13** [comment]
@Itsaadarsh You have been already assigned an issue.
@Devenkapoor7303 I can assign this to you but this depends on another issue. Looking forward to completion of that first

**Comment by Devenkapoor7303** [comment]
Can u tell me that first issue ??
Or can u assign this issue to me when that first issue get resolved??

**Comment by jackfrost13** [comment]
> Can u tell me that first issue ??
> Or can u assign this issue to me when that first issue get resolved??

The issue is referenced on the issue description. #152

**Comment by subhangi2731** [comment]
can you please assign it to me?

**Comment by priyanshu20** [comment]
@Devenkapoor7303 The dependent issue is closed, do u still want to take this up or shall I try?

**Comment by Devenkapoor7303** [comment]
U can try it .☺️
